en/docs/learn/handling-saml-single-logout-requests-from-federated-identity-providers.md

```diff
@@ -0,0 +1,147 @@
+# Handling SAML Single Logout Requests from Federated Identity Providers
+
+WSO2 Identity Server (WSO2 IS) supports handling SAML single logout requests from federated identity providers.
+After the SAML single logout request is received from the federated identity provider, Identity Server processes 
+the request and responds back to the identity provider after the session termination. This tutorial allows you to have
+hands-on experience on handling SAML single logout requests from the federated identity provider. 
+
+## Scenario
+
+Consider the following diagram where, WSO2 identity server, App 01 and App 02 are service providers of the federated 
+identity provider. App03 and App04 are service providers of wso2 identity server. When a user tries to logout
+from the App01, it will send a logout request to the federated IdP. Federated IdP will determine the session 
+participants using the session index available in the logout request. As the Identity server is a session participant,
+the Identity Server will receive a logout request from the federated IdP. Therefor the Identity Server needs to handle this
+request,terminate the session and response back with a valid logout response.
+
+
+![saml-slo-scenario](../assets/img/tutorials/saml-slo-scenario.png)
+        
+## Trying out the flow with WSO2 Identity Server
+
+To demonstrate the scenario, let's follow through the diagram given below.We are going to use two WSO2 IS servers which 
+are running on port 9443(Primary IS) and 9444 (Secondary IS) and two sample web applications called pickup-dispatch and 
+pickup-manager.
+
+![saml-slo-flow](../assets/img/tutorials/saml-slo-flow.png)
+
+The following sections guide you through configuring SAML single logout requests handling and trying it out with 
+the sample applications.
+
+-   Configuring Identity Provider(Secondary IS) in the Primary IS
+-   Configuring Pickup Dispatch application in the Primary IS
+-   Configuring Primary IS(Service Provider) in the Secondary IS
+-   Configuring Pickup Manager application in the Secondary IS
+
+### Configuring Identity Provider(Secondary IS) in the Primary IS
+
+1. Run the WSO2 Identity Server on port 9443 (Primary IS).
+
+2. Log in to the management console as an administration.
+
+3. Navigate to the Main menu to access the Identity menu. Click Add under Identity Providers. 
```

Comment:
Click **Identity>Identity Providers>Add** under the **Main** menu. 

Menu items, buttons, links, checkboxes need to be in bold. Please apply in all other instances throughout the document